modules/n1ql/pages/n1ql-language-reference/transactions-walkthrough.adoc

```diff
@@ -0,0 +1,249 @@
+= Use {sqlpp} with Couchbase Transactions
```

Comment:
Stash this change and use it to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)